draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -28,6 +29,7 @@ type Props = {
     }
   }
   persistent?: boolean
+  prominent?: boolean
```

Comment:
One pattern that we typically lean on when there could be future visual variants to components is to use a `variant` prop, rather than booleans. This [Paragraph component](https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/components/Paragraph/Paragraph.tsx#L43) is a great example of how we've done this in the past. 

The main motivation of doing it this way is to allow our components be flexible to future changes, and also making the component API a little easier to understand should more visual variants be added on in the future. Consider a future with multiple different visual styles of this block: 
```
type Props = {
   default?: boolean, 
   prominent?: boolean, 
   cautionary?: boolean, 
   positive?: boolean,
  /* other props */ 
} 
```

Each of these (theoretical) visual variants are exclusive, that is, you cannot have a GuidanceBlock that is a mixture of these props. So a `default` and `prominent`, or `positive` and `prominent` should not be mixed - however this approach allows consumers to mix. Having variant as a string forces the consumer to pick one visual variant. 

--- 

To be more specific, this could be approached like so:
```
type Props = {
   // we can add more variants in the future and it will be backwards compatible
   variant?: "default" | "prominent"
   /* other props */ 
} 
```

The existing users of the Guideance block will still receive the design they're expecting: 
```
const {
      actions: { primary, secondary },
      img,
      text,
      variant = "default", // 
      withActionButtonArrow
} = this.props
```